docs/src/main/tut/typeclasses/contravariantsemigroupal.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: docs
+title: "Contravariant Semigroupal"
+section: "typeclasses"
+source: "core/src/main/scala/cats/ContravariantSemigroupal.scala"
+scaladoc: "#cats.ContravariantSemigroupal"
+---
+# Contravariant Semigroupal
```

Comment:
We're kind of mixing the `Monoidal` and the `Semigroupal` in this doc. Maybe we should fix it all to be `ContravariantMonoidal`?

Reply:
Sounds good to me, I'll pin it all to `ContravariantMonoidal`.